diff -urN raine-0.51.12/source/savepng.c raine-0.51.12-patched/source/savepng.c
--- raine-0.51.12/source/savepng.c	2012-04-28 08:26:59.000000000 +1100
+++ raine-0.51.12-patched/source/savepng.c	2012-05-01 15:45:20.786274572 +1100
@@ -203,7 +203,7 @@
 	goto Error;
 
     /* Set error handling. */
-    if (setjmp(png_ptr->jmpbuf)) {
+    if (setjmp(png_jmpbuf(png_ptr))) {
 	/* If we get here, we had a problem reading the file. */
 	goto Error;
     }
